web/client/actions/geostory.js

```diff
@@ -172,3 +173,22 @@ export const move = (source, target, position) => ({
     target,
     position
 });
+
+/**
+ * Toggle focus on content
+ * @param {boolean} status if true set the focus otherwise remove
+ * @param {object} target the target content to be focused
+ * @param {object} section the story section parent of the target
+ * @param {object} parents all the target's ancestor, parent section excluded
```

Comment:
what is the purpose of `parents` and `section` ? I don't see it used anywhere else that in tests. Please remove unused variables, 
I see section has been used for background. see the comment about isBackground, the proposed solution should solve both the requirement